src/fsharp/lexhelp.fs

```diff
@@ -120,14 +120,14 @@ let stringBufferAsBytes (buf: ByteBuffer) =
     Array.init (bytes.Length / 2) (fun i -> bytes.[i*2]) 
 
 type LexerStringFinisher =
-    | LexerStringFinisher of (ByteBuffer -> LexerStringKind -> bool -> LexerContinuation -> token)
+    | LexerStringFinisher of (ByteBuffer -> LexerStringKind -> bool -> bool -> bool -> LexerContinuation -> token)
```

Comment:
-> bool -> bool -> bool ->  can you make this more usable?  Enum, flags enum or sumoesuch.